src/constants/solutionRuptureMap.ts

```diff
@@ -0,0 +1,197 @@
+import { LatLngExpression } from 'leaflet';
+
+interface SolutionAnalysisLocationOption {
+  id: string;
+  name: string;
+  latitude: number;
+  longitude: number;
+  population: number;
+}
+
+export const nz_centre: LatLngExpression = [-40.946, 174.167];
+export const zoom = 5;
+export const provider_url =
+  'https://server.arcgisonline.com/ArcGIS/rest/services/Ocean_Basemap/MapServer/tile/{z}/{y}/{x}';
+
+export const solutionRuptureMapRadiiOptions = {
```

Comment:
@0qy sorry this is not what I expected these lists should still be fetched from API. It when you fetch that needed to change.

Reply:
discussed and parked this for now